lib/wickedbase.pm

```diff
@@ -442,16 +442,11 @@ sub validate_macvtap {
     my ($self, $macvtap_log) = @_;
     my $ref_ip     = $self->get_ip(type => 'host',    netmask => 0, is_wicked_ref => 1);
     my $ip_address = $self->get_ip(type => 'macvtap', netmask => 0);
-    my $cmd_text   = "./check_macvtap $ref_ip $ip_address > $macvtap_log 2>&1 &";
-    sleep(30);    # OVS on a worker is slow sometimes to change and we haven't found better way how to handle it
-    type_string($cmd_text);
-    wait_serial($cmd_text, undef, 0, no_regex => 1);
-    type_string("\n");
-
+    script_run("./check_macvtap $ref_ip $ip_address > $macvtap_log 2>&1 & export CHECK_MACVTAP_PID=\$!");
```

Comment:
So you want to allow this to fail silently?

Reply:
In case it will fail , I will catch this here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/wickedbase.pm#L450